src/sentry/lang/native/symbolicator.py

```diff
@@ -321,7 +321,8 @@ def parse_sources(config):
         raise InvalidSourcesError(f"{e}")
 
     # remove App Store Connect sources (we don't need them in Symbolicator)
-    filter(lambda src: src.get("type") != "AppStoreConnect", sources)
+    if filter_appconnect:
+        filter(lambda src: src.get("type") != "AppStoreConnect", sources)
```

Comment:
```suggestion
        filter(lambda src: src.get("type") != SYMBOL_SOURCE_TYPE_NAME, sources)
```

can’t we use that for some reason? Also I’m wondering how this ever worked, since the type should be lower-camel-case, whereas you are matching with leading-camel-case here?

Reply:
lol, no this can't have worked.  But we can't import that constant because of circular imports.

Reply:
I've fixed the type name.  I guess another solution is to create a new module with just the constants that we need so we can import that everywhere.  I'm not really sure for now and am ok with either.  So if you strongly feel we should change this let me know, otherwise I'll leave it like this.

Reply:
I guess moving this and maybe some other leaf helpers/constants to a shared module to solve circular imports does make sense, but is not blocking here.